src/services/dispatchers.ts

```diff
@@ -76,7 +78,7 @@ async function synchronizeActions(sdk: OAuth1Client, localDispatcher: Dispatcher
 
       await dispatcherRepository.deleteAction(sdk, exhDispatcher.id, exhAction.id);
 
-      console.log(green(`Synchronized Action: ${exhAction.name} ✓`));
+      console.log(green(`Deleted Action: ${exhAction.name} ✓`));
```

Comment:
If we use `deleting` I would prefer to also use deleted instead of synchronized.